src/app/modules/footer/footer.component.html

```diff
@@ -3,37 +3,48 @@
     <div class="offers-container">
 
       <div class="offer-section">
-        <h2 class="offer-type">SALE OFFERS</h2>
-        <div class="offers-columns">
-          <div class="column" *ngFor="let column of saleOffersColumns">
-            <ng-container *ngFor="let offer of column">
-              <ng-container *ngComponentOutlet="offer"></ng-container>
-            </ng-container>
-          </div>
-        </div>
-      </div>
+  <h2 class="offer-type">SALE OFFERS</h2>
+  <div class="offers-columns">
+    <div class="column">
+      <app-sale-offers-button [brandId]="66" label="Jordan"></app-sale-offers-button>
```

Comment:
wszystkie te numerki trzeba wyciągnąc to jakiegoś stałego enuma typu Brands, nigdy nie robimy tak zwanych magic numbers. Link do tego co to jest magic numbers: https://en.wikipedia.org/wiki/Magic_number_(programming)